jena-text/src/test/resources/log4j.properties

```diff
@@ -12,5 +12,7 @@ log4j.logger.org.apache.jena.arq.exec=INFO
 # Reduce test/build noise
 log4j.logger.org.apache.jena.query.text.TextIndexLucene=ERROR
 
+# log4j.logger.org.apache.jena.query.text=TRACE
```

Comment:
Did you mean to leave this level of logging on for this PR?

Reply:
No I committed an update to log4j.properties @ 19:28Z to turn off the TRACE level.

Reply:
Cool!